mythicmaker/src/pages/Character.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useAuth } from './../AuthProvider';
+
+const Character = () => {
+  const { logout } = useAuth();
+
+  const handleLogout = () => {
+    logout();
```

Comment:
Jak onAuthStateChanged wyczyscisz usera, to nie bedziesz musiał tutaj tego robić.